hudi-test-suite/README.md

```diff
@@ -0,0 +1,300 @@
+[INFO] hudi-presto-bundle ................................. SUCCESS [  1.309 s]
+[INFO] hudi-utilities-bundle .............................. SUCCESS [ 18.386 s]
+[INFO] hudi-timeline-server-bundle ........................ SUCCESS [  8.600 s]
+[INFO] hudi-bench-bundle .................................. SUCCESS [ 38.348 s]
+[INFO] hudi-hadoop-docker ................................. SUCCESS [  2.053 s]
+[INFO] hudi-hadoop-base-docker ............................ SUCCESS [  0.806 s]
+[INFO] hudi-hadoop-namenode-docker ........................ SUCCESS [  0.302 s]
+[INFO] hudi-hadoop-datanode-docker ........................ SUCCESS [  0.403 s]
+[INFO] hudi-hadoop-history-docker ......................... SUCCESS [  0.447 s]
+[INFO] hudi-hadoop-hive-docker ............................ SUCCESS [  1.534 s]
+[INFO] hudi-hadoop-sparkbase-docker ....................... SUCCESS [  0.315 s]
+[INFO] hudi-hadoop-sparkmaster-docker ..................... SUCCESS [  0.407 s]
+[INFO] hudi-hadoop-sparkworker-docker ..................... SUCCESS [  0.447 s]
+[INFO] hudi-hadoop-sparkadhoc-docker ...................... SUCCESS [  0.410 s]
+[INFO] hudi-hadoop-presto-docker .......................... SUCCESS [  0.697 s]
+[INFO] hudi-integ-test .................................... SUCCESS [01:02 min]
+[INFO] ------------------------------------------------------------------------
+[INFO] BUILD SUCCESS
+[INFO] ------------------------------------------------------------------------
+[INFO] Total time: 04:23 min
+[INFO] Finished at: 2019-11-02T23:56:48-07:00
+[INFO] Final Memory: 234M/1582M
+[INFO] ------------------------------------------------------------------------
+```
+
+## Running on the cluster or in your local machine
+Copy over the necessary files and jars that are required to your cluster and then run the following spark-submit 
+command after replacing the correct values for the parameters. 
+NOTE : The properties-file should have all the necessary information required to ingest into a Hudi dataset. For more
+ information on what properties need to be set, take a look at the test suite section under demo steps.
+```
+shell$ ./prepare_integration_suite.sh --spark-command
+spark-submit --packages com.databricks:spark-avro_2.11:4.0.0 --master prepare_integration_suite.sh --deploy-mode
+--properties-file  --class org.apache.hudi.bench.job.HudiTestSuiteJob target/hudi-bench-0.5.1-SNAPSHOT.jar 
+--source-class  --source-ordering-field  --input-base-path  --target-base-path  --target-table  --props  --storage-type  --payload-class  --workload-yaml-path  --input-file-size  --<deltastreamer-ingest>
+```
+
+## Running through a test-case (local)
+Take a look at the TestHudiTestSuiteJob to check how you can run the entire suite using JUnit.
+
+## Running an end to end test suite in Local Docker environment
+
+```
+docker exec -it adhoc-2 /bin/bash
+# COPY_ON_WRITE tables
+=========================
+## Run the following command to start the test suite
+spark-submit \ 
```

Comment:
please start your docker and try to run both these commands after renaming and package changes to make sure they run fine..